nevergrad/functions/gym/multigym.py

```diff
@@ -557,6 +561,18 @@ def __init__(
 
         # Create the parametrization.
         parametrization = parameter.Array(shape=shape).set_name("ng_default")
+        if sparse_limit is not None:
+            parametrization1 = parameter.Array(shape=shape)
+            parametrization2 = parameter.Array(shape=shape)  # 0/1 weights for enabling/disabling
+            parametrization2.set_integer_casting()
+            parametrization2.set_bounds(0, 1)
```

Comment:
I'd be cautious here:
- setting bounds afterwards is not a very good idea: it uses the default sigma which is 1, while if you pass lower and upper in the init of Array it will use something better scaled and set the middle as initialization
- if everything is discrete, why not use ng.p.Choice([0, 1], repetitions=whatever)? this seems much better

Reply:
Ok. We lose the shape though.

Reply:
Hum there is a problem of shape. "repetitions" is necessarily an int.

Reply:
Ok fixed. The fact that the shape is lost is a bit an issue though.